src/AdvancedVI.jl

```diff
@@ -1,270 +1,122 @@
-module AdvancedVI
-
-using Random: Random
-
-using Distributions, DistributionsAD, Bijectors
-using DocStringExtensions
 
-using ProgressMeter, LinearAlgebra
-
-using ForwardDiff
-using Tracker
+module AdvancedVI
 
-const PROGRESS = Ref(true)
-function turnprogress(switch::Bool)
-    @info("[AdvancedVI]: global PROGRESS is set as $switch")
-    PROGRESS[] = switch
-end
+using SimpleUnPack: @unpack, @pack!
+using Accessors
 
-const DEBUG = Bool(parse(Int, get(ENV, "DEBUG_ADVANCEDVI", "0")))
+import Random: AbstractRNG, default_rng
```

Comment:
I guess
```suggestion
using Random: AbstractRNG, default_rng
```
is sufficient?